### PR TITLE
Derive Clone for AudioParam, could help the node bindings

### PR DESCRIPTION
**Pull request by orottier**
For motivation, see https://github.com/ircam-ismm/node-web-audio-api/pull/37

### CODE REVIEW

File: src/param.rs [hunk 1]
```diff
@@ -201,8 +201,9 @@ impl AudioParamEventTimeline {
 }
 
 /// AudioParam controls an individual aspect of an AudioNode's functionality, such as volume.
+#[derive(Clone)]
```
Comment (orottier) [inline src/param.rs:204]:
```suggestion
#[derive(Clone)] // for the node bindings, see #378
```

**Comment by b-ma** [comment]
Thanks!

**Comment by orottier** [comment]
Released in 0.36

**Pull request merged**